src/metadata/trackmetadata.h

```diff
@@ -114,6 +122,13 @@ class TrackMetadata {
         m_bitrate = bitrate;
     }
 
+    inline int getRating() const {
+        return m_rating;
+    }
+    inline void setRating(int rating) {
+        m_rating = rating;
```

Comment:
I do not have an overview of the effect of this changes. 

We should be aware of this Bug: 
Rating in Mixx != Rating in Mediaplayer
https://bugs.launchpad.net/mixxx/+bug/973136

What kind of rating is this here? 
Is it always the file rating tag? I yes the name should reflect this and we should have a comment that this is not necessarily the Mixxx rating.
Or should we go ahead and have two types of rating here?